src/Components/Recently-played-zone/Recently-played-zone.js

```diff
@@ -0,0 +1,26 @@
+import Card from '../Card/Card'
+import '../Style/Recently-played-zone.css'
+
+export const RecentlyPlayedZone = ({ receivedCards, addRecentCard }) => {
+    return (
+        <div className='recently-played-zone-container'>
+            <label className='recently-played-label'>Recently played</label>
+
+            <div className={`recently-played-cards-container-${receivedCards.length}`}>
+                {receivedCards.map((element) =>
+                    <Card
+                        key={element.id}
+                        id={element.id}
+                        title={element.title}
+                        imageUrl={element.url}
+                        description={element.description}
+                        addRecentCard={addRecentCard}
+                    />
+                )}
```

Comment:
you can use destructuring on cases like this, avoiding repeating every time the object name

```suggestion
                {receivedCards.map(({ id, title, url, description }) =>
                    <Card
                        key={id}
                        id={id}
                        title={title}
                        imageUrl={url}
                        description={description}
                        addRecentCard={addRecentCard}
                    />
                )}
```